storage/remote/queue_manager_test.go

```diff
@@ -362,6 +362,11 @@ func (c *TestStorageClient) waitForExpectedSampleCount() {
 }
 
 func (c *TestStorageClient) Store(_ context.Context, req []byte) error {
+	defer func() {
+		if r := recover(); r != nil {
+			fmt.Println("recovered")
+		}
+	}()
```

Comment:
This look spurious?

Reply:
Yep, was used for debugging a test failure. Removed for now, but will work on making test error more obvious in the future.